spec/system/dashboards/user_scrolls_down_dashboard_follows_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "rails_helper"
+
+RSpec.describe "Infinite scroll on dashboard", type: :system, js: true do
+  let(:user) { create(:user) }
+  let(:default_per_page) { 81 }
+  let(:users) { create_list(:user, default_per_page) }
+  let(:tags) { create_list(:tag, default_per_page) }
+  let(:organizations) { create_list(:organization, default_per_page) }
```

Comment:
this is another reason to have a parameterized `per_page`, so you can create 5 or 10 items, instead 81 per each type, it will considerably speed up the test.

Also, see if you can use `let_it_be` instead of `let`, you can do so if you're not modifying the object in the tests

Reply:
I get a weird error when I try to use ``let_it_be``, which only happens when I run the entire test suite. I'm not sure what's causing it, because it does not fail when I run my test alone. I'll use ``let`` for now and change it to ``let_it_be`` if I figure out what's causing this behavior.

```ruby
Failure/Error: let_it_be(:tags, reload: true) { create_list(:tag, default_per_page) }
     
     Algolia::AlgoliaProtocolError:
       Cannot reach any host: 

  ================================================================================

       An HTTP request has been made that VCR does not know how to handle:
         PUT https://zox5q9ty1r.algolia.net/1/indexes/Tag_test/79174
     
       There is currently no cassette in use. There are a few ways
       you can configure VCR to handle this request:
     
         * If you're surprised VCR is raising this error
           and want insight about how VCR attempted to handle the request,
           you can use the debug_logger configuration option to log more details [1].
         * If you want VCR to record this request and play it back during future test
           runs, you should wrap your test (or this portion of your test) in a
           `VCR.use_cassette` block [2].
         * If you only want VCR to handle requests made while a cassette is in use,
           configure `allow_http_connections_when_no_cassette = true`. VCR will
           ignore this request since it is made when there is no cassette [3].
         * If you want VCR to ignore this request (and others like it), you can
           set an `ignore_request` callback [4].
     
       [1] https://www.relishapp.com/vcr/vcr/v/5-0-0/docs/configuration/debug-logging
       [2] https://www.relishapp.com/vcr/vcr/v/5-0-0/docs/getting-started
       [3] https://www.relishapp.com/vcr/vcr/v/5-0-0/docs/configuration/allow-http-connections-when-no-cassette
       [4] https://www.relishapp.com/vcr/vcr/v/5-0-0/docs/configuration/ignore-request
```

Reply:
It's possible that it's a precedence problem, `let_it_be` is basically a before all: https://test-prof.evilmartians.io/#/let_it_be

Maybe you can get away with the `let!` ?